client/src/components/GroupCard/GroupCard.js

```diff
@@ -0,0 +1,21 @@
+
+import React from 'react';
+import "./GroupCard.css";
+
+const GroupCard = (props) => {
+
+  let classList = `GroupCard`;
```

Comment:
If there's just the one class, then let's remove the extra variable and put it in the `className` attribute